### PR TITLE
Add Product Development page

### LINKED ISSUE #862: [Services] Add page for Product Development
**This issue is part of website revamp project**
simplabs website improvements aim to increase the ease of updating content, adding new pages, enhance content readability, and layout responsiveness.

## What needs to be done
A new page needs to be added under the 'Digital Products' section. The page called "Product Development" will contain  all the information related to the product development offering as laid out on the following mockup:

https://share.goabstract.com/b69a3eca-2239-4585-b2f4-d7c81e52c76d?collectionLayerId=3863e3f2-b71b-4f8b-9c0d-8c8cc936b003&mode=design

### Styling
The page will require additional styling in order to implement the clipped photos.

- [x] Add Page

- [x] Update Content

### PR DESCRIPTION
**Pull request by marcoow**
This adds the page for product development. It **only** adds the page and text content – styling and addition of images etc. is still needed to be done in a follow-up PR.

localhost_4200_services_product-development

closes #862

### CODE REVIEW

File: src/ui/components/PageProductDevelopment/template.hbs [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<div>
+  <Header
+    @active="services"
+    @title="Product Development"
+    @documentTitle="Product Development | Services"
+    @description="From start to finish you work with the same team of product experts, engineers and designers to bring your product to live."
+  >
+    <HeaderContent @label="Product Development" @headline="Engineering excellence means building things to last">
```
Comment (locks) [inline src/ui/components/PageProductDevelopment/template.hbs:8]: I'm not sure if single quote cuts it here or if it should be smart quotes.

```suggestion
    <HeaderContent @label="Product Development" @headline="Engineering excellence means 'building things to last'">
```

File: src/ui/components/PageProductDevelopment/template.hbs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      <p class="typography.lead">
+        simplabs has its origins in engineering and development. We obsess over crafting the absolute best products we can, built on architectures that are easy to extend, update and maintain. Our process depends on close collaboration between designers and developers to release well-tested products in dynamic, fast-paced iterations.
```
Comment (locks) [inline src/ui/components/PageProductDevelopment/template.hbs:10]:
```suggestion
        Simplabs has its origins in engineering and development. We obsess over crafting the absolute best products we can, built on architectures that are easy to extend, update and maintain. Our process depends on close collaboration between designers and developers to release well-tested products in dynamic, fast-paced iterations.
```

File: src/ui/components/PageProductDevelopment/template.hbs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    </p>
+    <ArrowLink @href="/contact/">
+      Talk to an engineer
+    </ArrowLink>
+  </div>
+  <div class="layout.full offset.after-12">
+    <h2>
+      Excellence in engineering, from first commit to finished product
+    </h2>
+    <p class="typography.lead">
```
Comment (locks) [inline src/ui/components/PageProductDevelopment/template.hbs:58]: Should we create a card to style this list as in the mockup?

File: src/ui/components/PageProductDevelopment/template.hbs [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        Intensive automated testing and QA
+      </li>
+      <li>
+        Engineering Team of Technology Leaders
+      </li>
+      <li>
+        Deployment and Operation
+      </li>
+    </ul>
+  </div>
```
Comment (locks) [inline src/ui/components/PageProductDevelopment/template.hbs:84]: "Learn How We Work" link missing

File: src/ui/components/PageProductDevelopment/template.hbs [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  <div class="layout.full offset.after-12">
+    <h2>
```
Comment (locks) [inline src/ui/components/PageProductDevelopment/template.hbs:86]: The mockup does not include this section @ghislaineguerin, does it need updating?

**Review by locks — CHANGES_REQUESTED** [review]
Left some inline comments.

File: src/ui/components/PageProductDevelopment/template.hbs (hunk 2 above)
Comment (marcoow) [reply src/ui/components/PageProductDevelopment/template.hbs:10]: _"simplabs"_ is always lowercase

File: src/ui/components/PageProductDevelopment/template.hbs (hunk 3 above)
Comment (marcoow) [reply src/ui/components/PageProductDevelopment/template.hbs:58]: So my understanding was these issues are about adding the pages and text content **only**?

File: src/ui/components/PageProductDevelopment/template.hbs (hunk 5 above)
Comment (marcoow) [reply src/ui/components/PageProductDevelopment/template.hbs:86]: This text is in Basecamp - I think it's missing in the mockup?

File: src/ui/components/PageProductDevelopment/template.hbs (hunk 4 above)
Comment (marcoow) [reply src/ui/components/PageProductDevelopment/template.hbs:84]: added

File: src/ui/components/PageProductDevelopment/template.hbs (hunk 1 above)
Comment (marcoow) [reply src/ui/components/PageProductDevelopment/template.hbs:8]: I left that out intentially as I think it's a bit strange. If we want quotes I think we should be using proper ones but then what do we say by putting it in quotes anyway - to me that would indicate we're implying that nothing ever lasts or so which I'm not sure why we want to be implying that.

File: src/ui/components/PageProductDevelopment/template.hbs (hunk 3 above)
Comment (ghislaineguerin) [reply src/ui/components/PageProductDevelopment/template.hbs:58]: @locks @marcoow I think we should create a card to style the lists.

File: src/ui/components/PageProductDevelopment/template.hbs (hunk 5 above)
Comment (ghislaineguerin) [reply src/ui/components/PageProductDevelopment/template.hbs:86]: I will check and update the mockup if needed.

File: src/ui/components/PageProductDevelopment/template.hbs (hunk 3 above)
Comment (marcoow) [reply src/ui/components/PageProductDevelopment/template.hbs:58]: I think I'm not clear what the intention of the issue is. Is is about adding the page and content only or also styling it? If so, can we do that without even having the shapes for the images etc., e.g. will the card not need to change potentially once we have the image shapes?

Anyway, I don't think we want a component to style this but a CSS block that we can apply?

**Review by locks — APPROVED** [review]

**Review by ghislaineguerin — APPROVED** [review]

**Pull request merged**